pkg/store/proxy_test.go

```diff
@@ -2295,3 +2295,154 @@ func TestDedupRespHeap_Deduplication(t *testing.T) {
 	}
 
 }
+
+func TestDedupRespHeap_QuorumChunkDedup(t *testing.T) {
+	t.Parallel()
+
+	for _, tcase := range []struct {
+		responses []*storepb.SeriesResponse
+		testFn    func(responses []*storepb.SeriesResponse, h *responseDeduplicator)
+		tname     string
+	}{
+		{
+			tname: "edge case with only one response",
```

Comment:
one response is valid from `store` component IIUC?

Reply:
Good point. This name is copied from an existing test case.